nteract_coffee_sample.ipynb

```diff
@@ -0,0 +1,299 @@
+              "['yes', 'yes', 'yes', 'yes', 'yes', 'yes', 'yes', 'yes', 'yes', 'yes']"
+            ]
+          },
+          "metadata": {}
+        }
+      ],
+      "execution_count": 8,
+      "metadata": {
+        "collapsed": false,
+        "outputHidden": false,
+        "inputHidden": false
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "captain.add_pip_packages(\"nbconvert\")"
+      ],
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "You've already launched your coffee boat. I'll add this package at runtime using magic, but next time add your packages before so I don't have to use my magical super powers (aka make your code slow).\n",
+            "Writing req file to /tmp/coffee_boat_tmp_mogVbx/magicCoffeeReq94CAtS.\n",
+            "Estimated number of executors is 4\n",
+            "Installed package on remote\n",
+            "Installing package locally\n"
+          ]
+        }
+      ],
+      "execution_count": 9,
+      "metadata": {
+        "collapsed": false,
+        "outputHidden": false,
+        "inputHidden": false
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "def foo2(x):\n",
+        "    import nbconvert\n",
+        "    return \"yes\"\n",
+        "rdd.map(foo2).collect()"
+      ],
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "execution_count": 10,
+          "data": {
+            "text/plain": [
+              "['yes', 'yes', 'yes', 'yes', 'yes', 'yes', 'yes', 'yes', 'yes', 'yes']"
+            ]
+          },
+          "metadata": {}
+        }
+      ],
+      "execution_count": 10,
+      "metadata": {
+        "collapsed": false,
+        "outputHidden": false,
+        "inputHidden": false
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "sc.stop()"
+      ],
+      "outputs": [],
+      "execution_count": 11,
+      "metadata": {
+        "collapsed": false,
+        "outputHidden": false,
+        "inputHidden": false
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [],
+      "outputs": [],
+      "execution_count": 12,
+      "metadata": {
+        "collapsed": false,
+        "outputHidden": false,
+        "inputHidden": false
+      }
+    }
+  ],
+  "metadata": {
+    "kernelspec": {
+      "name": "python_spark_testing_sadness",
+      "language": "python",
+      "display_name": "python_spark_testing_sadness"
+    },
+    "kernel_info": {
+      "name": "python_spark_testing_sadness"
+    },
+    "language_info": {
+      "mimetype": "text/x-python",
+      "nbconvert_exporter": "python",
+      "name": "python",
+      "pygments_lexer": "ipython2",
+      "version": "2.7.6",
+      "file_extension": ".py",
+      "codemirror_mode": {
+        "version": 2,
+        "name": "ipython"
+      }
+    },
+    "nteract": {
+      "version": "0.4.3"
```

Comment:
You should upgrade your nteract version, v0.6.3 is pretty nice. :wink: It's probably not auto-upgrading because you're on the Linux version (IIRC).